simoc_server/agent_model/agent_model.py

```diff
@@ -347,22 +362,35 @@ def init_agents(self, model):
 
         atmosphere = AgentModel.create_atmosphere(model, structures)
         plumbing_system = AgentModel.create_plumbing_system(model, structures)
+
         model.add_agent(atmosphere)
         model.add_agent(plumbing_system)
         for i in range(self.NUM_HUMANS):
             model.add_agent(agents.HumanAgent(model, structure=crew_quarters))
 
         # TODO determine number of plants for base line model
-        for i in range(self.NUM_CABBAGE):
-            model.add_agent(agents.CabbageAgent(model, structure=greenhouse))
+        for i in range(self.NUM_PEANUT):
+            model.plants_available.append(agents.PeanutAgent(model))
 
-        for i in range(self.NUM_CARROTS):
-            model.add_agent(agents.CarrotAgent(model, structure=greenhouse))
+        for i in range(self.NUM_SOYBEAN):
+            model.plants_available.append(agents.SoybeanAgent(model))
```

Comment:
This will cause issues, storing an instance of the plant and reusing it means the same instance gets added to the model repeatedly with the same unique_id